.github/workflows/build.yml

```diff
@@ -6,7 +6,9 @@ on:
     - '*'
   push:
     branches: [ MK3, MK3_* ]
-
+    tags:
+    - 'v*'
+    
```

Comment:
Minor thing. white spaces

Reply:
I'm not sure I follow. Diff shows a blank line removed on the left, it's readded below on the right